### PR TITLE
website: Variables page update

### PR DESCRIPTION
**Pull request by gretanausedaite**
Updated values in Variables page.

### CODE REVIEW

**Review by mayank99 — APPROVED** [review]

File: apps/website/src/pages/docs/variables.mdx [hunk 1]
```diff
@@ -31,17 +31,17 @@ import '@itwin/itwinui-variables';
 
 The main background colors increase in darkness from 1 to 5.
```
Comment (FlyersPh9) [inline apps/website/src/pages/docs/variables.mdx:32]:
```suggestion
```

The line about 1 to 5 can be removed.

File: apps/website/src/pages/docs/variables.mdx [hunk 2]
```diff
@@ -50,27 +50,27 @@ better contrast in high-contrast themes.
...
+    'var(--iui-color-border-accent)',
+    'var(--iui-color-border-positive)',
+    'var(--iui-color-border-warning)',
+    'var(--iui-color-border-negative)',
   ]}
 />
 
 ### Foreground colors
 
 The foreground colors also increase in darkness from 1 to 5.
```
Comment (FlyersPh9) [inline apps/website/src/pages/docs/variables.mdx:64]:
```suggestion
```
The line about 1 to 5 can be removed here as well.

**Review by FlyersPh9 — APPROVED** [review]
Once the 2 comments I left are resolved, this looks good to me.  Approving in advance.

File: apps/website/src/pages/docs/variables.mdx (hunk 1 above)
Comment (elephantcatdog) [reply apps/website/src/pages/docs/variables.mdx:32]: You could change it to something like "There are several main background colors." to keep the format of "text then bullet points".

**Pull request merged**